src/products/focus/how_to_localize.md

```diff
@@ -0,0 +1,21 @@
+# Focus - How to localize
+
+[Focus for Android](https://pontoon.mozilla.org/projects/focus-for-android/) and [Focus for iOS](https://pontoon.mozilla.org/projects/focus-for-ios/) are both on a four week rapid release cycle. This means that usually there are new releases going out every four weeks.
+
+Within this timeframe, localizers will have approximately four weeks to localize and test their strings for a given release: strings land continuously on Nightly during the first four weeks, and then these strings ride the train to the Beta version.
+
+It is important to use the Nightly versions for both products, in order to test localization work regularly (see more details about testing [here](testing_focus.md)).
+
+## Localize Focus for Android and Focus for iOS in Pontoon
+
+Localizers should go to the [Focus for Android page on Pontoon](https://pontoon.mozilla.org/projects/focus-for-android/) - or the [Focus for iOS page on Pontoon](https://pontoon.mozilla.org/projects/focus-for-ios/) - and click "Request New Language". They will be notified by email once the locale is set up and the project is ready to be localized.
+
+When the locale has been set up, localizers can choose it from the list of locales on Pontoon, from the same page they made the locale request (links above).
```

Comment:
The second part sounds a bit off to me (maybe `in the same page from which they made the locale request`)